### DISCUSSION TITLE
cookies blocked using auth0

### DISCUSSION THREAD

**Discussion by nseb** [ionic-team/capacitor#3171] [Q&A]
### Capacitor Version
  @capacitor/cli: 2.2.0

  @capacitor/core: 2.2.0

  @capacitor/android: 2.2.0

  @capacitor/electron: 2.2.0

  @capacitor/ios: 2.2.0

### Platform(s)
Android / IOS

Hello,
I transform my pwa in Capacitor App (with quasar) , I use auth0 authentification , and I must set https. It’s worked ,I can login. But I relaunch application , I have a problem cookie :
A cookie associated with a cross-site resource at https://xxxx.eu.auth0.com/ was set without the SameSite attribute. It has been blocked,

My pwa use paypal , and i have the same problem :
A cookie associated with a cross-site resource at https://paypal.com/ was set without the SameSite attribute. It has been blocked

any idea ?

**Comment by IT-MikeS** [comment]
What are you using to make the requests?

**Comment by nseb** [comment]
Paypal very simple , my main html page contains : 
<script src="https://www.paypal.com/sdk/js?client-id=xxxx"/>
For auth0 , I use this dependencies : 
https://auth0.com/docs/libraries/auth0-spa-js

I use quasar framework to transform my pwa to app mobile

**Comment by snpori** [comment]
Same problem here. it seems like the cookies are broken.
Are you trying with capacitor-community/http ?
